### ISSUE TITLE
Support simple visualization of (expression) data

### ISSUE THREAD

**Issue by mruffalo** [hubmapconsortium/hubmap-data-portal#10]
Implement a simple expression heatmap at the level of tissues/organs. This visualization should gain many other features over time, but we should start with something straightforward that we can (finish) implement(ing) ASAP.

The human anatomical diagram in production at https://demo1.hubmapconsortium.org/ currently re-colors the organs/tissues after page load, with random values from the REST API. Adjust this to:
1. only re-color the diagram after a user searches for/selects a gene
2. use real data and not random values
3. adjust the color of the organ/tissue outlines in addition to the "fill" color

**Comment by mruffalo** [comment]
We can implement this with a very simple data model for a first prototype, just adding a (single) foreign key from `Gene` to a new `TissueExpressionHeatmap` class which stores colors for each tissue.

For a given gene, the data in its `TissueExpressionHeatmap` should be shown in a full-page visualization, with the same header and footer as the main page, showing the same anatomical diagram, but with elements replacing the charts. We should show the gene name large and bold to the right of the diagram, e.g. in something like a `<h1>`, and maybe a bar chart of expression across tissues. Not sure whether we should display the data twice though: as colors in the anatomical diagram, and numerically in the bar chart.

Maybe (and I really mean "maybe"; I'm still getting familiar with React) this could be structured by having the main application use this layout:

```
<Header ... />
<PageContent ... />
<Footer ... />
```

with a `PageContent` component that doesn't yet exist, which would act as a container that holds the sequence of `<Grid/>` elements which currently comprise the main page.

Displaying the "gene expression by tissue" page might then involve replacing the `<PageContent ... />` element.

**Comment by sushma-ananda** [comment]
@mruffalo Thanks for adding this. Yes, as we discussed this morning, we have RootContainer which is as you mentioned Grid Containers with grid items, where I want to add this. I am presently working on search gene => change colors of tissues where the genes are expressed. Do you think we can split this into multiple tasks ? and can we assign priority to tasks here ? Best, Sushma.

**Comment by mruffalo** [comment]
Sounds great -- do you mean something like separate GitHub issues, or a checkbox list of subtasks here?

**Comment by sushma-ananda** [comment]
Yes, since this task came up only yesterday noon, I just want to make sure, I promise something that make it work for example: replacing PageContent . I would definitely attempt for sure. So for tasks that come up last minute, if we separate most important functionality from "maybe" functionality, will be helpful. what do you think about this ? for example, I have completed search by gene by searching from backend and reflecting it on human image as of now, but there are more things to do. so if we decide to demo only: search by gene, show/reflect on human image and then update charts, table -> if this is primary goal: it is acheivable, hopefully before next week. But if we want to do a) show a home page that consists of human figure, the two charts we discussed yesterday with Ziv and then the table as well as b) search by gene and updated charts + table. Both a) and b) dont seem 100% feasible just for next week by Tuesday. note: there are a bunch of things: that are not uploaded/demoable since I am checking on a live JS tools to work this through.  I am not quiet sure if I can work over this weekend unlike past few weekends when I have been working. Can you please suggest what features are priority for next week?

**Comment by mruffalo** [comment]
Priority 1 would be implementing a search by gene, getting the results from the Django backend, and then recoloring the anatomical figure with the results. This could be as simple as performing this query/recoloring after the user types a gene name into the search box and presses Enter, without changing anything else on the page or implementing additional views. This should probably replace the recoloring that happens on page load.

Showing the results in a separate page is nowhere near as important -- and showing the results in a bar graph is even lower priority.

It would be fine if the only thing implemented was "type gene name, press Enter, tissues are recolored with values from backend".

**Comment by sushma-ananda** [comment]
Sure Matt. Here is the status so far: 1) search by gene on human image: gets all tissues relevant to the gene searched and using tissue-gene heatmap , fills the color of tissues only relevant to the gene searched and shows an animation. Status of 1) is completed. Please let me know if you prefer any changes for step 1.  my next attempt is to show (group by tissue name -> institution -> stacked bar graph as we discussed  and this to display this summary on graph. So then when user searches for the gene (besides updating human image, should filter the chart and possibly table). This is what I am working on right now, does this sound good to you?
